web/src/lib.rs

```diff
@@ -373,6 +374,11 @@ impl Ruffle {
             .unwrap_or(JsValue::NULL)
     }
 
+    pub fn get_actually_used_renderer(&self) -> JsValue {
```

Comment:
I have changed its name to `renderer_name` and will push once I have tested it.

In regards to its return type, I am calling that function from Typescript, so I do not know whether it can have a different type than `JsValue` .

Reply:
Can you give it a try at least? The `wasm-bindgen` docs doesn't mention whether returning `&str` is supported or not, but I do see `String` is documented: https://rustwasm.github.io/docs/wasm-bindgen/reference/types/string.html

Reply:
Sorry, I tried, but I know too little of Rust to figure out the types.

Reply:
This seems to be https://github.com/rustwasm/wasm-bindgen/issues/1187. A workaround would probably be returning a `String`, but I think this can be skipped for now.